### PR TITLE
chore(client): replace ncc with esbuid

### PR DESCRIPTION
**Pull request by timsuchanek**

### CODE REVIEW

File: src/.buildkite/test/run.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -ex
 
 cd src
 
-npm i --silent -g pnpm@5.1.7
+npm i --silent -g pnpm@5.1.7 esbuild@0.7.1 --unsafe-perm
```
Comment (Jolg42) [inline src/.buildkite/test/run.sh:7]: You're using --unsafe-perm here but not in the publish one?

Comment (timsuchanek) [reply src/.buildkite/test/run.sh:7]: Oops

**Comment by Jolg42** [comment]
Why did you add a 0 byte dev.db? https://github.com/prisma/prisma/blob/esbuild/src/packages/client/src/__tests__/integration/dev.db

**Comment by Jolg42** [comment]
Could you update the CONTRIBUTING.md with this `esbuild@0.7.1 --unsafe-perm` ?

File: src/packages/client/package.json [hunk 2]
```diff
@@ -37,19 +37,16 @@
...
     "generate": "node scripts/postinstall.js",
     "postinstall": "node scripts/postinstall.js",
-    "ncc:runtime": "node scripts/build-runtime.js",
-    "cleanup": "rm -f index.d.ts && cp scripts/backup-index.js index.js && cp scripts/backup-index.d.ts index.d.ts",
-    "ncc:generator": "rm -rf generator-build && ncc build dist/generator.js -o generator-build --minify --source-map && rm -f generator-build/index.js.bak && chmod +x generator-build/index.js",
-    "prepare": "pnpm run cleanup && cp scripts/backup-index.js index.js && cp scripts/backup-index.d.ts index.d.ts && node scripts/download.js || echo \"\"",
-    "prepublishOnly": "pnpm run build"
+    "prepare": "cp scripts/backup-index.js index.js && cp scripts/backup-index.d.ts index.d.ts && node scripts/download.js || echo \"\"",
+    "prepublishOnly": "pnpm run build",
+    "patch-package": "patch-package"
```
Comment (Jolg42) [inline src/packages/client/package.json:49]: I guess you can remove `patch-package`

**Comment by Jolg42** [comment]
Why was this db file updated? 🤔 
https://github.com/prisma/prisma/blob/1627a5474dde486955ac59f103bd28ad38c9ea72/src/packages/client/fixtures/blog/prisma/dev.db

**Comment by timsuchanek** [comment]
> Why was this db file updated? 🤔
> [`1627a54`/src/packages/client/fixtures/blog/prisma/dev.db](https://github.com/prisma/prisma/blob/1627a5474dde486955ac59f103bd28ad38c9ea72/src/packages/client/fixtures/blog/prisma/dev.db)

Not yet sure tbh

**Pull request merged**